chromium_src/chrome/browser/prefs/sources.gni

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//components/gcm_driver/config.gni")
+
+# Fix: Added import loop. Need this import for using |enable_widevine| vars.
```

Comment:
as mentioned in DM we should import brave/browser/sources.gni directly in the existing patch for chrome/browser/BUILD.gn which should help with other circular include issues cc @mariospr

Reply:
Fixed by https://github.com/brave/brave-core/pull/8460/commits/461f70c943aa984819aa3cf394a7f7dedd5b392a